src/Sulu/Bundle/WebsiteBundle/Tests/Functional/Controller/SitemapControllerTest.php

```diff
@@ -79,62 +102,59 @@ public function testIndexMultipleLanguage(): void
 
         $this->assertCount(2, $crawler->filterXPath('//x:urlset/x:url'));
 
-        $this->assertEquals('http://test.lo/en-us', $crawler->filterXPath('//x:urlset/x:url[1]/x:loc')->text());
+        $this->assertEquals('http://test.lo/en/test-1-0', $crawler->filterXPath('//x:urlset/x:url[1]/x:loc')->text());
         $this->assertEquals(
-            'en-us',
+            'en',
             $crawler->filterXPath('//x:urlset/x:url[1]/xhtml:link[1]')->attr('hreflang')
         );
         $this->assertEquals(
-            'http://test.lo/en-us',
+            'http://test.lo/en/test-1-0',
             $crawler->filterXPath('//x:urlset/x:url[1]/xhtml:link[1]')->attr('href')
         );
         $this->assertEquals(
-            'en',
+            'en-us',
             $crawler->filterXPath('//x:urlset/x:url[1]/xhtml:link[2]')->attr('hreflang')
         );
         $this->assertEquals(
-            'http://test.lo/en',
+            'http://test.lo/en-us/test-1-0',
             $crawler->filterXPath('//x:urlset/x:url[1]/xhtml:link[2]')->attr('href')
         );
 
-        $this->assertEquals('http://test.lo/en', $crawler->filterXPath('//x:urlset/x:url[2]/x:loc')->text());
+        $this->assertEquals('http://test.lo/en-us/test-1-0', $crawler->filterXPath('//x:urlset/x:url[2]/x:loc')->text());
         $this->assertEquals(
-            'en',
+            'en-us',
             $crawler->filterXPath('//x:urlset/x:url[2]/xhtml:link[1]')->attr('hreflang')
         );
         $this->assertEquals(
-            'http://test.lo/en',
+            'http://test.lo/en-us/test-1-0',
             $crawler->filterXPath('//x:urlset/x:url[2]/xhtml:link[1]')->attr('href')
         );
         $this->assertEquals(
-            'en-us',
+            'en',
             $crawler->filterXPath('//x:urlset/x:url[2]/xhtml:link[2]')->attr('hreflang')
         );
         $this->assertEquals(
-            'http://test.lo/en-us',
+            'http://test.lo/en/test-1-0',
             $crawler->filterXPath('//x:urlset/x:url[2]/xhtml:link[2]')->attr('href')
         );
-
-        $this->assertEquals('http://test.lo/en', $crawler->filterXPath('//x:urlset/x:url[2]/x:loc')->text());
     }
 
-    public function testIndexMultipleTlds(): void
```

Comment:
the controller is not responsible for this only the providers itself. So its not sitemap controller test which should test this. the page sitemap provider tests need take care of that which was already tackled in the new bundle.